vision/google/cloud/vision/_http.py

```diff
@@ -48,8 +49,9 @@ def annotate(self, image, features):
         data = {'requests': [request]}
         api_response = self._connection.api_request(
             method='POST', path='/images:annotate', data=data)
-        responses = api_response.get('responses')
-        return responses[0]
+        images = api_response.get('responses')
+        if len(images) == 1:
+            return Annotations.from_api_repr(images[0])
```

Comment:
Yeah, so it shouldn't be an error actually. I think I should return a list of the images. I was planning on doing that in a soon following PR.

I'll have this raise an exception for now if you like and then I'll take it back out with my image batching PR.